unlock-app/src/services/storageService.ts

```diff
@@ -121,7 +133,7 @@ export class StorageService extends EventEmitter {
    * @param {string} password (do not send to locksmith)
    * @returns {Promise<*>}
    */
-  async createUser(user, emailAddress, password) {
+  async createUser(user: any, emailAddress: string, password: string) {
     const opts = {}
     return axios.post(`${this.host}/users/`, user, opts)
     // return {
```

Comment:
We don't do anything with email and password arguments in the original function. Any reason?